packages/x-date-pickers/src/models/adapters.ts

```diff
@@ -112,7 +112,6 @@ export interface MuiPickersAdapter<TDate> {
   date(value?: any): TDate | null;
   /**
    * Convert a date in the library format into a JavaScript `Date` object.
-   * @deprecate Will be removed in v7.
```

Comment:
It could also be done with `parseISO`/`toISO`

I agree to only keep one source of truth to only keep one way to exchange between the adapter and the `Date` object. Is their one easier to manipulate respectively to TimeZone management?